pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py

```diff
@@ -237,12 +237,23 @@ def _stop_request_generator(self, request_generator):
         Args:
             request_generator (Generator): A streaming pull request generator
                 returned from :meth:`_request_generator_thread`.
+            response_generator (grpc._channel._Rendezvous): The gRPC
```

Comment:
The type you're looking for is `grpc.Call`.

Reply:
I can change it if you want, but I feel like that's me yielding to the insanity that is `_Rendezvous`, so I am not keen on it.

Reply:
I'd really prefer it, and actually, to add to the madness this is actually [`grpc.Future`](https://grpc.io/grpc/python/grpc.html#grpc.Future).